### ISSUE TITLE
Capstone project submission

### ISSUE THREAD

**Issue by b3nharris** [ds4owd-001/project-b3nharris#3]
Hi @larnsce @sskorik01 @mianzg,

Two questions on the project which I think only you can answer:

1. **Submission Date**. I'm away from 25 Jan - 18 Feb. I'm going to get a draft project in before I leave, but I think it may need some tidying to make sure it meets all the requirements. Would it be ok if I had an extension until the end of February to address any outstanding issues.
2. **Style**. I was planning to write the project in more the style of a blog - it's more appropriate to the analysis I'm doing and the message I want to send. I think I can merge this with the requirements of the assignment (references, citations, etc) but the narrative will be a little different. Are you ok with that?

Thanks- will join office hours on Thursday if I've made progress on my project.

Ben

**Comment by larnsce** [comment]
Thanks @b3nharris, for the questions. I will take this over from @sskorik01, who we had noted down as supervising your project.

> Submission Date

Yes, that's okay. What date would be comfortable for you to submit?

> Style

Perfect, I am really excited to see the outcome. 

See you Thursday, maybe.

**Comment by b3nharris** [comment]
Hi @larnsce,

Sorry for not replying sooner. For the submission, would a date of 26 February be ok? That would give me a full week after I get back to finalise the project.

I've made progress on project:

- I have all my data imported and in the format I want now (I did some work to simplify the code I'd already written for this.
- I've done proof of concept plots for the visualisations I want
- I've done a rough draft of the intro and methods.

Still a lot to do to finish the analysis and add the items to meet the requirements, but I think the basics are in place so it should be straightforward. I don't have any technical questions for now.

Thanks,
Ben
